mteb/tasks/Classification/ara/OnlineStoreReviewSentimentClassification.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+N_SAMPLES = 2048
+
+class OnlineStoreReviewSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="OnlineStoreReviewSentimentClassification",
+        dataset={
+            "path": "Ruqiya/Arabic_Reviews_of_SHEIN",
+            "revision": "8ea114aa27b82a52373203830dc2f5b540b6fcac",
+        },
+        description="This dataset contains Arabic reviews of products from the SHEIN online store.",
+        reference="https://huggingface.co/datasets/Ruqiya/Arabic_Reviews_of_SHEIN",
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs=["ara-Arab"],
+        main_score="accuracy",
+        date=("2024-05-01", "2024-05-15"),
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="apache-2.0",
+        socioeconomic_status="mixed",
+        annotations_creators="derived",
+        dialect=["ara-arab-EG", "ara-arab-JO", "ara-arab-LB", "ara-arab-SA"],
```

Comment:
@KennethEnevoldsen which standard are we using for dialects?

Reply:
`HotelReviewSentimentClassification`, `TweetEmotionClassification` and a few others seem to follow similar representation.

@Ruqyai Are these dialects mentioned somewhere or is this guessed?

Reply:
I saw how to write `HotelReviewSentimentClassification` as a guide and added the same thing since they are reviews written in Arabic and I think they both will be the same way.

Reply:
I think it is better to add only `"ara-arab-SA"` dialect of Saudi Arabia as I see most of reviews 
What do you think?

Reply:
Correct?

**ISO 639-2  ara
ISO 15924 Arab
ISO 3166-1 alpha-2  SA**

`"ara-Arab-SA"`
@KennethEnevoldsen

Reply:
yea indeed it does seem to be correct